WorkloadManagementSystem/Utilities/test/Test_ParametricJob.py

```diff
@@ -127,6 +156,20 @@ def test_Multi(self):
     self.assertEqual( jobClassAd.getAttributeString( 'Arguments' ), '3 b' )
     self.assertEqual( jobClassAd.getAttributeString( 'JobName' ), 'Test_1' )
 
+  def test_MultiBad(self):
+
+    clad = ClassAd( TEST_JDL_MULTI_BAD )
+    result = getParameterVectorLength( clad )
+    self.assert_( not result['OK'] )
+
+  def test_NoParameters(self):
+
+    clad = ClassAd( TEST_JDL_NO_PARAMETERS )
+    result = getParameterVectorLength( clad )
+    self.assert_( result['OK'] )
+    nParam = result['Value']
+    self.assert_( nParam is None )
```

Comment:
assert_ is deprecated. use assertIsNone
https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsNone